R/runvb.R

```diff
@@ -102,7 +102,7 @@ eagle.helper = function(alt,n,xFull,xNull,s){
   stopifnot( sapply(alt,length)==sapply(n,length), sapply(n,length)==sapply(xFull,nrow), sapply(xFull,nrow)==sapply(xNull,nrow) )
 
   stopifnot( all(sapply(alt,class)  %in% c("integer","numeric") ), all(sapply(n,class) %in% c("integer","numeric") ) )
-  stopifnot( all(sapply(xFull,class)=="matrix" ), all(sapply(xNull,class)=="matrix") )
+#  stopifnot( all(sapply(xFull,class)=="matrix" ), all(sapply(xNull,class)=="matrix") )
```

Comment:
The [recommended fix](https://developer.r-project.org/Blog/public/2019/11/09/when-you-think-class.-think-again/) to update this for R4 is to use `inherits()` instead of `==`